rails-blog/app/sequent/post/post.rb

```diff
@@ -22,5 +26,8 @@ def initialize(command)
     on PostContentChanged do |event|
       @content = event.content
     end
+
+    on PostDestroyed do
```

Comment:
Perhaps you already know: You don't need to add this handler if you don't do anything with event in your state.

Reply:
I figured, but I didn't want to diverge from the other examples in this file.